src/compiler/parser.ts

```diff
@@ -6814,6 +6817,69 @@ namespace ts {
                     return finishNode(result);
                 }
 
+                function parseAuthorTag(start: number, tagName: Identifier): JSDocAuthorTag {
+                    const result = <JSDocAuthorTag>createNode(SyntaxKind.JSDocAuthorTag, start);
+                    result.tagName = tagName;
+
+                    const comment = tryParse(() => tryParseAuthorNameAndEmail());
+
+                    if (comment) {
+                        result.comment = comment;
+                    }
+
+                    return finishNode(result);
+                }
+
+                function tryParseAuthorNameAndEmail(): string | undefined {
+                    const comments: string[] = [];
+                    let seenLessThan = false;
+                    let seenGreaterThan = false;
+                    let seenAtToken = false;
+                    let token = scanner.getToken();
+
+                    loop: while (true) {
+                        switch (token) {
+                            case SyntaxKind.Identifier:
+                            case SyntaxKind.WhitespaceTrivia:

```

Comment:
this looks like it will include whitespace just after the initial tag, with `@author X Y` resulting in the comment `" X Y"`. Is that correct?

Reply:
No, it doesn't do that. I tried with arbitrary amount of spaces/tabs and it ignored it all until the first non-whitespace character.

See the baseline for the quick info test which doesn't include the single whitespace, which is following after the `author` tag.

Test tag
https://github.com/Microsoft/TypeScript/blob/7deee5d21a28504ee7cf4dcf53ed41c7be5c16d2/tests/cases/fourslash/quickInfoJsDocTags.ts#L6
Parsed tag
https://github.com/Microsoft/TypeScript/blob/7deee5d21a28504ee7cf4dcf53ed41c7be5c16d2/tests/baselines/reference/quickInfoJsDocTags.baseline#L73